UI/src/view/linear_asset/speedLimitStyle.js

```diff
@@ -144,7 +164,7 @@
     historyStyle.addRules(oneWayOverlayStyleRules);
 
     var isUnknown = function(speedLimit) {
-      return !_.isNumber(speedLimit.value);
+      return _.isUndefined(speedLimit.value) || !_.isNumber(speedLimit.value.value);
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code